packages/azure-http-specs/specs/azure/core/page/mockapi.ts

```diff
@@ -48,3 +48,44 @@ Scenarios.Azure_Core_Page_listWithCustomPageModel = passOnSuccess({
   response: { status: 200, body: json({ items: [validUser] }) },
   kind: "MockApiDefinition",
 });
+
+Scenarios.Azure_Core_Page_withParameterizedNextLink = passOnSuccess([
+  {
+    // First page request
+    uri: "/azure/core/page/parameterized-next-link",
```

Comment:
```suggestion
    uri: "/azure/core/page/with-parameterized-next-link",
```

![image](https://github.com/user-attachments/assets/b8792e32-be63-4379-973b-08797c80586e)

Reply:
Maybe it's the definition that's having issue? 
Should be `@route("parameterized-next-link")`.